views/album.ejs

```diff
@@ -1,4 +1,5 @@
 <link href="/<%= urlRoot %>/gallery.css" rel="stylesheet" type="text/css">
+<div class="breadcrumb">
```

Comment:
Good idea - worried about bootstrap users seeing ill effects, but not sure if it applies on a div...